keras_cv/layers/preprocessing/color_jitter.py

```diff
@@ -0,0 +1,158 @@
+        and lower bound number. If factor is a single value, the range will 
+        be `(0, saturation_factor)`; otherwise `(saturation_factor[0], saturation_factor[1])`
+        as lower and upper bound respectively. It will be used to randomly pick a value 
+        from uniform distribution. Default: (0.5, 0.9).
+
+        hue_factor: A non-negative single float value. It will be used 
+        as `[-hue_factor, hue_factor)` to randomly pick a value from uniform 
+        distribution. Default: 0.5.
+
+        seed: Integer. Used to create a random seed. Default: None.
+
+    Call arguments: 
+        images: Tensor representing images of shape
+            `(..., height, width, channels)`, with dtype tf.float32 / tf.uint8, or,
+            `(height, width, channels)`, with dtype tf.float32 / tf.uint8
+        training: A boolean argument that determines whether the call should be run 
+            in inference mode or training mode. Default: True.
+   
+    Usage:
+    ```python
+    (images, labels), _ = tf.keras.datasets.cifar10.load_data()
+    color_jitter = keras_cv.layers.ColorJitter()
+    augmented_images = color_jitter(images)
+    ```
+    """
+
+    def __init__(
+        self,
+        brightness_factor=0.5,
+        contrast_factor=(0.5, 0.9),
+        saturation_factor=(0.5, 0.9),
+        hue_factor=0.5,
+        seed=None,
+        **kwargs,
+    ):
+        super().__init__(**kwargs)
+        self.seed = seed
+        self.brightness_factor = self._check_factor_limit(
+            brightness_factor, name="brightness"
+        )
+        self.contrast_factor = self._check_factor_limit(
+            contrast_factor, name="contrast"
+        )
+        self.saturation_factor = self._check_factor_limit(
+            saturation_factor, name="saturation"
+        )
+        self.hue_factor = self._check_factor_limit(hue_factor, name="hue")
+
+    def _check_factor_limit(self, factor, name):
+        if isinstance(factor, (int, float)):
+            if factor < 0:
+                raise TypeError(
+                    "The factor value should be non-negative scalar or tuple "
+                    f"or list of two upper and lower bound number. Received: {factor}"
+                )
+            if name == "brightness" or name == "hue":
+                return abs(factor)
+            return (0, abs(factor))
+        elif isinstance(factor, (tuple, list)) and len(factor) == 2:
+            if name == "brightness" or name == "hue":
+                raise ValueError(
+                    "The factor limit for brightness and hue, it should be a single "
+                    f"non-negative scaler. Received: {factor} for {name}"
+                )
+            return sorted(factor)
+        else:
+            raise TypeError(
+                "The factor value should be non-negative scalar or tuple "
+                f"or list of two upper and lower bound number. Received: {factor}"
+            )
+
+    def _color_jitter(self, images):
```

Comment:
I think we should save the standardized param in the layer (either [-1, 1] or [0, 1]), and then convert them to the value that API needs in the layer body here.

Reply:
(repeat of same point as above: https://github.com/keras-team/keras-cv/blob/master/.github/API_DESIGN.md#strength-parameters)

Please provide a docstring explanation of the conversion.